style.css

```diff
@@ -1,265 +1,275 @@
+@import "../libraries/style.css";
```

Comment:
The Wordpress Codex talks about parent themes being loaded via functions.php - see MITLibraries/courtyard-blog#23 for more info